desktop/core/src/desktop/api2_tests.py

```diff
@@ -37,6 +37,113 @@ def setUp(self):
     self.user = User.objects.get(username="api2_user")
 
 
+  def test_import_document_with_forward_ref(self, client=None):
+    if client is None:
+      client = self.client
+    
+    doc = '''[\n{\n  "model": "desktop.document2",\n  "pk": 20,\n  "fields": {\n    "owner": [\n      "admin"\n    ],\n'''\
```

Comment:
What are we testing here? Is it possible to keep `doc` to its minimum needed for the test? That will make the intention of the test clearer and easier to debug when it fails.

Reply:
this PR is only for the testing purpose, I want to just show that forward ref is failing for py2 and passing for python 3.
I will delete this PR after merging the https://github.com/cloudera/hue/pull/3157.